packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+			"options": [
+				{
+					"label": "Room sensor",
+					"value": 1
+				},
+				{
+					"label": "Floor sensor",
+					"value": 2
+				},
+				{
+					"label": "Room + Floor sensor",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "11",
+			"label": "Room Temperature Correction",
+			"description": "Correct room temperature sensor. Allowable range: -10 - 10",
+			"valueSize": 1,
+			"minValue": -10,
+			"maxValue": 10,
+			"unit": "0.5 °C",
+			"defaultValue": 0
+		},
+
+		{
+			"#": "12",
+			"label": "Floor Temperature Correction",
+			"description": "Correct floor temperature sensor. Allowable range: -10 - 10",
+			"valueSize": 1,
+			"minValue": -10,
+			"maxValue": 10,
+			"unit": "0.5 °C",
+			"defaultValue": 0
+		},
+
+		{
+			"#": "13",
+			"label": "Drying Mode Duration",
+			"description": "Allowable range: 5-100",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 100,
+			"unit": "minutes",
+			"defaultValue": 30
+		},
+
+		{
+			"#": "14",
+			"label": "Mode After Drying",
+			"description": "Set the mode after drying mode completes.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off mode",
+					"value": 0
+				},
+				{
+					"label": "Manual mode",
+					"value": 1
+				},
+				{
+					"label": "Energy save mode",
+					"value": 2
+				},
+				{
+					"label": "Away mode",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "16",
+			"label": "Temperature Report on Value",
+			"description": "When to report temperature relative change actively. Allowable range: 0 (disable), 2-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "0.1 °C",
+			"defaultValue": 2
+		},
+
+		{
+			"#": "17",
+			"label": "Temperature Report on Interval",
+			"description": "Time interval to report temperature. Allowable range: 0 (disable), 30-28800",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 28800,
+			"unit": "seconds",
+			"defaultValue": 300
+		},
+
+		{
+			"#": "18",
+			"label": "Power Change Threshold",
+			"description": "Threshold when to report power change. Allowable range: 0 (disable), 1-100",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"unit": "W",
+			"defaultValue": 5
+		},
+
+		{
+			"#": "19",
+			"label": "Current Change Threshold",
+			"description": "Threshold when to report current change. Allowable range: 0 (disable), 1-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "0.1 A",
+			"defaultValue": 10
+		},
+
+		{
+			"#": "20",
+			"label": "Voltage Change Threshold",
+			"description": "Threshold when to report voltage change. Allowable range: 0 (disable), 1-10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"unit": "V",
+			"defaultValue": 2
+		},
+
+		{
+			"#": "21",
+			"label": "Meter Report Interval",
+			"description": "Time interval for active report of Meter. Allowable range: 60-2678400",
```

Comment:
Unnecessary description. No need to restate the range if it matches min/maxValue.